### ISSUE TITLE
Using Annotation

### ISSUE THREAD

**Issue by Gaurabp** [legalthings/angular-pdfjs-viewer#45]
Can anyone help me out how to use annotation on it like- comments,underline,drawing,etc.

**Comment by jasny** [comment]
@Gaurabp I don't think that is something that is supported by [Mozilla's PDF.js viewer](https://mozilla.github.io/pdf.js/web/viewer.html). If it is, and is just missing from this angular plugin, please show an example.
